docs/modules/ROOT/pages/features.adoc

```diff
@@ -1,19 +1,43 @@
 = Project Syn Features
```

Comment:
For me this is a very abstract reading and I can imaging readers to easily drop out not getting why all this is awesome. To be hones, I have no idea how to make it better.

Reply:
Some parts are a bit of a bumpy read. E.g _… the catalog GitOps Git repository …_, _… Tenant and Cluster Kubernetes-CRD configuration objects …_. I understand the intention of being precise. Maybe this can be improved somehow.

Reply:
Yeah, I can see what you mean. But I can't come up with something "better". Therefore I close this comment...